src/nsyt/metadata.rs

```diff
@@ -159,11 +170,11 @@ impl ImageMetadata for Image {
     }
 
     fn get_mast_az(&self) -> Option<f64> {
-        None
+        unimplemented!()
```

Comment:
Already merged, but am putting these back to `None`. `unimplemented!()` will cause a panic when the metadata is written back out to disk, which is done in a mission-agnostic manner and uses all fields. `mast_az` and `mast_el` don't apply to InSight and will always be none.